src/main/java/core/basesyntax/Storege.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storege {
+    public static Map<String, Integer> fruitStorage = new HashMap<>();
+    public static Map<String, LocalDate> expirationDateStorage = new HashMap<>();
```

Comment:
you may rename this class `StorageService` and implement the necessary methods here

the storage may remain the static field
I would only recommend to try to make it a single collection of items

you may want to read the task again to understand the storage logic